openedx/core/djangoapps/content_libraries/collections/handlers.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
Since these handlers aren't actually doing anything yet, can this file be deleted?
We'll likely have the handlers live with `content/search/handlers.py`, since that's the app that cares about these events.

Reply:
Good point, I mainly had it there to confirm that the signals were actually being sent, removed them here: [a68f4e2](https://github.com/openedx/edx-platform/pull/35321/commits/a68f4e2a46c6ff784f465a945b4ceb0125b94234)